docs/docs/user-guide/using-proxy-url.md

```diff
@@ -0,0 +1,21 @@
+# Configure CLI for Microsoft 365 to use Proxy
+
+If you are behind a corporate proxy, you may need to configure the CLI for Microsoft 365 to use a proxy server to access Microsoft 365 services. You can configure a proxy URL using the [cli config set](/docs/docs/cmd/cli/config/config-set.md) command.
+
+## Understanding Proxy URL
+
+Before configuring the proxy, it's important to understand the different parts of a proxy URL. A proxy URL typically consists of the following elements:
+
+- **Protocol**: The protocol used by the proxy server, such as HTTP, HTTPS, or SOCKS.
+- **Username and Password**: If the proxy server requires authentication, you will need to provide a username and password.
+- **Host**: The hostname or IP address of the proxy server.
+- **Port number**: The port number on which the proxy server is listening. Defaults to 80 when not provided.
+
+## Configuring Proxy URL
+To configure CLI for Microsoft 365 to use a proxy, you need to execute the m365 cli config set command with the following syntax:
```

Comment:
Let's add an empty line here to properly separate the heading from the paragraph.